com.unity.multiplayer.mlapi/Runtime/Logging/FileLogger.cs

```diff
@@ -0,0 +1,38 @@
+using System.IO;
+
+namespace MLAPI.Logging
+{
+    /// <summary>
+    /// Helper class for logging
+    /// Saves to a file identified by the process ID of each client
+    /// This is useful when debugging what happens on multiple machines
+    /// ! Only recommended for debugging, not for production !
```

Comment:
rather than just calling this out, should we wrap this into `#if DEVELOPMENT_BUILD || UNITY_EDITOR` ?

Reply:
Good point ! will do.